mvpa2/datasets/miscfx.py

```diff
@@ -537,3 +537,39 @@ def plot(self):
         pl.xlabel('Offset')
         pl.ylabel('Correlation Coefficient')
         pl.show()
+
+
+@datasetmethod
+def to_df(ds, sas=None, fas=None):
```

Comment:
@mih if your are still using pymvpa from time to time, feedback on this little helper would be appreciated.